app/src/main/java/com/antonioleiva/bandhookkotlin/ui/screens/detail/DetailActivity.kt

```diff
@@ -55,6 +56,8 @@ class DetailActivity : BaseActivity(), DetailView, ScrollableHeaderActivity,
 
         supportActionBar?.setDisplayHomeAsUpEnabled(true)
         supportsLollipop { image.transitionName = BaseActivity.IMAGE_TRANSITION_NAME }
+
+        findViewById(android.R.id.content).setTransitionGroupCompat(true)
```

Comment:
I'm almost sure you can use `transitionName` in XML, old versions will just ignore it, and you will save some lines of code. Did you try it?